CHANGELOG.md

```diff
@@ -1,5 +1,94 @@
 # CHANGELOG
 
+## 4.7.00
+
+[Full Changelog](https://github.com/kokkos/kokkos/compare/4.6.02...4.7.00)
+
+### Features:
+
+### Backend and Architecture Enhancements:
+
+#### CUDA:
+* Support RDC with Clang 17+ and use new offload driver [\#7831](https://github.com/kokkos/kokkos/pull/7831)
+* Add support for AMPERE87 architecture (Jetson Orin Nano) [\#8092](https://github.com/kokkos/kokkos/pull/8092)
+* Directly use fp16 mathematical function when available instead of casting back and forth to fp32 [\#7698](https://github.com/kokkos/kokkos/pull/7698)
+
+#### HIP:
+
+#### SYCL:
+- Allow requesting out-of-order queues [\#8091](https://github.com/kokkos/kokkos/pull/8091)
+- Allow using non-trivially-copyable comparators with oneDPL [\#7538](https://github.com/kokkos/kokkos/pull/7538)
+- Directly use fp16 mathematical function when available instead of casting back and forth to fp32 [\#7698](https://github.com/kokkos/kokkos/pull/7698)
+
+#### OpenMPTarget:
+- Remove support for non-llvm compilers [\7973](https://github.com/kokkos/kokkos/pull/7973)
+
+#### OpenACC:
+
+#### HPX:
+
+#### Threads:
+
+#### OpenMP:
+
+#### Serial:
+
+### General Enhancements
+* Add support for Zen 5 AMD microarchitecture [\#7877](https://github.com/kokkos/kokkos/pull/7877)
+* Add support for NVIDIA's Blackwell GPU architecture [\#7888](https://github.com/kokkos/kokkos/pull/7888)
```

Comment:
Wasn't this added in 4.6.1?
```suggestion
```

Reply:
Yes, looks like this was already added into 4.6.1. Removed.